test/deploy.actions.test.js

```diff
@@ -120,6 +120,30 @@ test('deploy full manifest', async () => {
   expect(runtimeLibUtils.syncProject).toHaveBeenCalledWith('sample-app-1.0.0', global.sampleAppConfig.manifest.src, expectedDistManifest, mockEntities, { fake: 'ow' }, expect.anything(), undefined, true)
 })
 
+test('deploy full manifest with OpenWhishk Auth Handler', async () => {
+  addSampleAppFiles()
+  runtimeLibUtils.processPackage.mockReturnValue(deepCopy(mockEntities))
+
+  const buildDir = global.sampleAppConfig.actions.dist
+  // fake a previous build
+  const fakeFiles = {}
+  fakeFiles[path.join(buildDir, 'action.js')] = 'fakecontent'
+  fakeFiles[path.join(buildDir, 'action-zip.zip')] = 'fake-content'
+  global.fakeFileSystem.addJson(fakeFiles)
+
+  let sampleConfigWithAuthHandler = structuredClone(global.sampleAppConfig)
+  sampleConfigWithAuthHandler.ow.auth_handler = 'fake:BearerToken'
```

Comment:
small change - don't make this a string. reflect the actual value it would be so people scanning the tests know what is expected here:
```
{
  getAuthHandler: async () => {}
}
```